packages/core/src/connectivity/scp-cf/destination/destination-accessor-types.ts

```diff
@@ -29,9 +29,12 @@ export interface DestinationAccessorOptions {
   userJwt?: string;
 
   /**
-   * @hidden
+   * This property is only considered in case no userJwt is provided.
+   * It is meant for situations where you do not have a token e.g. background processes.
+   * The value for iss is the issuer field of a JWT e.g. https://<your-subdomain>.localhost:8080/uaa/oauth/token'
+   *
+   * ATTENTION: If this property is used no validation of the provided subdomain value is done in contrast to the JWT case.
```

Comment:
I found this sentence a bit slippery, not sure if this is an improvement:
```suggestion
   * ATTENTION: If this property is used, no validation of the provided subdomain value is done. This is differs from how the `userJwt` is handled.
```